pallets/proxy-rmrk-equip/src/lib.rs

```diff
@@ -322,13 +471,15 @@ impl<T: Config> Pallet<T> {
 				collection.id,
 				token_id,
 				PropertyScope::Rmrk,
-				<PalletCore<T>>::rmrk_property(EquippableList, &part.equippable)?,
+				<PalletCore<T>>::encode_rmrk_property(EquippableList, &part.equippable)?,
 			)?;
 		}
 
 		Ok(())
 	}
 
+	/// Ensure that the collection under the Base ID is a Base collection,
+	/// and fetch it.
```

Comment:
Is it possible I have missed some explanation of what "Base", and "Base ID" is? Because for me these terms are not very clear

Reply:
Yeah, it's in the `Terminology` and `Proxy Implementation` in the header :)